github/files.go

```diff
@@ -0,0 +1,42 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/google/go-github/v64/github"
+)
+
+func SelectFile(file *github.CommitFile) *github.CommitFile {
+	// file filename contains dashboard in the name return file
```

Comment:
Is this comment relevant?

Reply:
In go you should add comments to all your functions as these comments get added to the package page